testsuite/cluster-test/src/main.rs

```diff
@@ -763,7 +763,7 @@ impl ClusterTestRunner {
 
     async fn wait_until_all_healthy(&mut self) -> Result<()> {
         info!("Waiting for all validators to be healthy");
-        let wait_deadline = Instant::now() + Duration::from_secs(20 * 60);
+        let wait_deadline = Instant::now() + Duration::from_secs(5 * 60);
```

Comment:
just concern the shorten duration is enough for node recovery?

Reply:
Yeah, from all practical purposes recovery happens much faster than 5 minutes.